examples/rgbpp/local/launch/2-launch-rgbpp.ts

```diff
@@ -0,0 +1,98 @@
+import { Collector, buildRgbppLockArgs, genRgbppLaunchCkbVirtualTx, RgbppTokenInfo, appendCkbTxWitnesses, updateCkbTxWithRealBtcTxId, sendCkbTx } from '@rgbpp-sdk/ckb';
+import { DataSource, ECPair, bitcoin, NetworkType, sendRgbppUtxos, transactionToHex } from '@rgbpp-sdk/btc';
+import { BtcAssetsApi, BtcAssetsApiError } from '@rgbpp-sdk/service';
+import { RGBPP_TOKEN_INFO } from './0-rgbpp-token-info';
+
+// BTC SECP256K1 private key
+const BTC_TEST_PRIVATE_KEY = '0000000000000000000000000000000000000000000000000000000000000001';
+// API docs: https://btc-assets-api.testnet.mibao.pro/docs
+const BTC_ASSETS_API_URL = 'https://btc-assets-api.testnet.mibao.pro';
+// https://btc-assets-api.testnet.mibao.pro/docs/static/index.html#/Token/post_token_generate
+const BTC_ASSETS_TOKEN = '';
+
+interface Params {
+  ownerRgbppLockArgs: string;
+  launchAmount: bigint;
+  rgbppTokenInfo: RgbppTokenInfo;
+}
+const launchRgppAsset = async ({ ownerRgbppLockArgs, launchAmount, rgbppTokenInfo }: Params) => {
+  const collector = new Collector({
+    ckbNodeUrl: 'https://testnet.ckb.dev/rpc',
+    ckbIndexerUrl: 'https://testnet.ckb.dev/indexer',
+  });
+  const isMainnet = false;
+
+  const network = isMainnet ? bitcoin.networks.bitcoin : bitcoin.networks.testnet;
+  const keyPair = ECPair.fromPrivateKey(Buffer.from(BTC_TEST_PRIVATE_KEY, 'hex'), { network });
+  const { address: btcAddress } = bitcoin.payments.p2wpkh({
+    pubkey: keyPair.publicKey,
+    network,
+  });
+
+  console.log('btc address: ', btcAddress);
+
+  const networkType = isMainnet ? NetworkType.MAINNET : NetworkType.TESTNET;
+  const service = BtcAssetsApi.fromToken(BTC_ASSETS_API_URL, BTC_ASSETS_TOKEN, 'https://btc-test.app');
```

Comment:
```suggestion
  const service = BtcAssetsApi.fromToken(BTC_ASSETS_API_URL, BTC_ASSETS_TOKEN, BTC_ASSETS_API_ORIGIN);
```

Perhaps the `https://btc-test.app` can also be defined as a constant

Reply:
Yes. That's a good idea and I will update all the examples later